documentation/src/test/java/org/hibernate/search/documentation/mapper/orm/indexing/Author.java

```diff
@@ -31,6 +32,8 @@
 	@OneToMany(mappedBy = "author")
 	private List<Book> books = new ArrayList<>();
 
+	private LocalDate birthday;
```

Comment:
I can't help it, I'm going to be pedantic (well, more than I usually am). While I understand what you mean, technically this would be the "birthDate". See https://en.wiktionary.org/wiki/birthday:

> birthday (plural birthdays)
> 1. The anniversary of the day on which someone is born.
> 2. The anniversary of the day on which something is created.
> 3. The date on which someone is born or something is created, **more commonly called birthdate** or date of birth.
> 4. A birthday party

Reply:
Yes. I meant `birthDate`.